examples/src/main/java/ai/djl/examples/inference/ObjectDetectionWithTensorflowSavedModel.java

```diff
@@ -0,0 +1,250 @@
+
+        Path imagePath = outputDir.resolve("detected-tensorflow-model-dog_bike_car.png");
+        // OpenJDK can't save jpg with alpha channel
+        newImage.save(Files.newOutputStream(imagePath), "png");
+        logger.info("Detected objects image has been saved in: {}", imagePath);
+    }
+
+    static Map<Integer, String> loadSynset() throws IOException {
+        URL synsetUrl =
+                new URL(
+                        "https://raw.githubusercontent.com/tensorflow/models/master/research/object_detection/data/mscoco_label_map.pbtxt");
+        Map<Integer, String> map = new ConcurrentHashMap<>();
+        int maxId = 0;
+        try (InputStream is = synsetUrl.openStream();
+                Scanner scanner = new Scanner(is)) {
+            scanner.useDelimiter("item ");
+            while (scanner.hasNext()) {
+                String content = scanner.next();
+                content = content.replaceAll("(\"|\\d)\\n\\s", "$1,");
+                Item item = JsonUtils.GSON.fromJson(content, Item.class);
+                map.put(item.id, item.displayName);
+                if (item.id > maxId) {
+                    maxId = item.id;
+                }
+            }
+        }
+        return map;
+    }
+
+    private static final class Item {
+        int id;
+
+        @SerializedName("display_name")
+        String displayName;
+    }
+
+    private static final class MyTranslator implements Translator<Image, DetectedObjects> {
+
+        private Map<Integer, String> classes;
+        private int maxClassId;
+        private int maxBoxes;
+        private float threshold;
+
+        MyTranslator() {
+            maxBoxes = 10;
+            threshold = 0.7f;
+        }
+
+        @Override
+        public NDList processInput(TranslatorContext ctx, Image input) {
+            // input to tf object-detection models is a list of tensors, hence NDList
+            NDArray array = input.toNDArray(ctx.getNDManager(), Image.Flag.COLOR);
+            // optionally resize the image for faster processing
+            array = NDImageUtils.resize(array, 224);
+            // tf object-detection models expect 8 bit unsigned integer tensor
+            array = array.toType(DataType.UINT8, true);
+            array = array.expandDims(0); // tf object-detection models expect a 4 dimensional input
+            return new NDList(array);
+        }
+
+        @Override
+        public void prepare(NDManager manager, Model model) throws IOException {
+            if (classes == null) {
+                classes = loadSynset();
+                maxClassId = Collections.max(classes.keySet());
+            }
+        }
+
+        @Override
+        public DetectedObjects processOutput(TranslatorContext ctx, NDList list) {
+            // output of tf object-detection models is a list of tensors, hence NDList in djl
+            // output NDArray order in the list are not guaranteed
+            int length = 0; // number of boxes in the output
```

Comment:
FYI: The output order is random especially during debug. Have to do some magic here.